grafana-grafonnet.yaml

```diff
@@ -29,12 +29,40 @@ pipeline:
       install -m755 *.json "${{targets.destdir}}"/opt/jsonnet/grafonnet/
 
       for i in clean custom raw; do
-        mkdir -p ${{targets.destdir}}/opt/jsonnet/grafonnet/$i
-        mv $i "${{targets.destdir}}"/opt/jsonnet/grafonnet/$i/
+        if [ -d "$i" ]; then
+          cp -r "$i" "${{targets.destdir}}"/opt/jsonnet/grafonnet/
+        fi
       done
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-bitnami-compat
+    description: "compat package for bitnami/grafonnet library"
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: grafana-operator
+          version-path: 5/debian-12
```

Comment:
I think we should also need to crate a `-bitnami-compat` in the `grafana-operator` package. Upstream image has those dirs:

![Screenshot 2025-02-27 at 18 23 47](https://github.com/user-attachments/assets/06f6d612-b8ba-4f40-955b-e5c5f2facc34)